src/main/java/org/apache/cassandra/sidecar/config/yaml/RestoreJobConfigurationImpl.java

```diff
@@ -36,6 +36,10 @@ public class RestoreJobConfigurationImpl implements RestoreJobConfiguration
     public static final int DEFAULT_JOB_DISCOVERY_RECENCY_DAYS = 5;
     public static final int DEFAULT_PROCESS_MAX_CONCURRENCY = 20; // process at most 20 slices concurrently
     public static final long DEFAULT_RESTORE_JOB_TABLES_TTL_SECONDS = TimeUnit.DAYS.toSeconds(90);
+    public static final String RESTORE_JOB_LONG_RUNNING_HANDLER_THRESHOLD_SECONDS =
+    "restore_job_long_running_threshold_seconds";
+    // A restore job handler is considered long-running if it has been in the "active" list for 10 minutes.
+    private static final long DEFAULT_RESTORE_JOB_LONG_RUNNING_HANDLER_THRESHOLD_SECONDS = 600;
```

Comment:
Let's use TimeUnit to define the time for consistency and accuracy
```suggestion
    private static final long DEFAULT_RESTORE_JOB_LONG_RUNNING_HANDLER_THRESHOLD_SECONDS = TimeUnit.MINUTES.toSeconds(10);
```

Reply:
So I tried this while implementing it, but it ends up that using a non-constant value means you can't use it as the default value as part of the @JsonProperty annotations... so I'd end up having to put `"600"` in the `@JsonProperty` if we used `TimeUnit.MINUTES.toSeconds(10)` here. 😢